css/global.css

```diff
@@ -469,6 +469,15 @@ Utility Classes ( flexbox, wrappers,  etc.)
 .section-header {
 	padding: var(--spacing-lg);
 	text-align: center;
+	gap: var(--spacing-lg);
+}
+
+.gradient-header {
+	background: var(--grad-blue-purple);
+}
+
+.gradient-header h2 {
+	color: var(--color-white);
```

Comment:
there is a text color assignment you can apply to text or their containers `--text-color-light` or `text-color-dark` respectively